src/app/src/components/Dashboard.jsx

```diff
@@ -50,11 +51,7 @@ function Dashboard({
     }
 
     if (!userWithAccessHasSignedIn) {
-        return (
-            <AppGrid title={DASHBOARD_TITLE}>
-                Unauthorized
-            </AppGrid>
-        );
+        return <RouteNotFound />;
```

Comment:
👍 to showing Not Found here

I am seeing a flash of the Dashboard title and a spinner before not found loads. We can move the resolution of this behavior to a separate bug issue.

![2019-06-19 14 04 44](https://user-images.githubusercontent.com/17363/59800383-61ce3180-929b-11e9-9289-db08d30f70a2.gif)

Reply:
Interesting! I imagine this is probably due to the order in which the rendering conditions are sequenced so I'll see if it's easy to fix.

Reply:
This was in fact a sequence issue -- there was a block of code that rendered the "Dashboard" title when the session sign in was underway. I fixed it in 1a383689c050bb58159e4208dde798ddf8e59a65 by removing the title.